src/ServicePulse.Host.Tests/Api/APIApprovals.cs

```diff
@@ -1,13 +0,0 @@
-﻿using NUnit.Framework;
-using Particular.Approvals;
-using System.IO;
-
-class APIApprovals
-{
-    [Test]
-    public void PlatformSampleApprovals()
```

Comment:
Why isn't this necessary anymore?

Reply:
Because: 
`//HINT: If this test fails the Particular.PlatformSample project's app.constants.js probably needs to be updated`

Is no longer relevant. There will no longer need to be a separate `app.constants.js` in Particular.PlatformSample, so it can't get out of sync.